.github/workflows/rust.yml

```diff
@@ -308,6 +308,22 @@ jobs:
       - name: cargo clippy
         run: cargo xclippy -D warnings
 
+  move-clippy:
+    needs: diff
+    if: needs.diff.outputs.isRust == 'true'
+    runs-on: [ubuntu-ghcloud]
+    steps:
+      - uses: arduino/setup-protoc@v1
+        # this avoids rate-limiting
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v3
+      - run: rustup component add clippy
+      - name: cargo move-clippy
+        run: |
+          cd external-crates/move
+          cargo move-clippy -D warnings
```

Comment:
Why is this copied here and in `external.yml`?

Reply:
The other clippy one was in both places, and I was just trying to pattern match for updates. Which is the better place to have it live?